backend/src/main/java/ca/bc/gov/hlth/mohums/controller/UsersController.java

```diff
@@ -333,14 +287,14 @@ boolean isAuthorizedToViewClient(String token, String clientGuid) {
 
     }
 
-    private MultiValueMap<String, String> buildQueryEventActiveParam (String start, String nbElementMax, Optional<String> dateFrom, Optional<String> dateTo){
-    
-            MultiValueMap<String, String> queryEventParams = new LinkedMultiValueMap<>();
-            queryEventParams.add("type", "LOGIN");
-            queryEventParams.add("first", start);
-            queryEventParams.add("max", nbElementMax);
-            dateFrom.ifPresent(dateFromValue -> queryEventParams.add("dateFrom", dateFromValue));
-            dateTo.ifPresent(dateToValue -> queryEventParams.add("dateTo", dateToValue));
-            return queryEventParams;
+    private MultiValueMap<String, String> buildQueryEventActiveParam (int start, int nbElementMax, Optional<String> dateFrom, Optional<String> dateTo){
+
+        MultiValueMap<String, String> queryEventParams = new LinkedMultiValueMap<>();
+        queryEventParams.add("type", "LOGIN");
+        queryEventParams.add("first", String.valueOf(start));
+        queryEventParams.add("max", String.valueOf(nbElementMax));
```

Comment:
In the main function body we want to deal with `int`, so I push the conversion down to the code that actually cares, which is setting the query parameters.